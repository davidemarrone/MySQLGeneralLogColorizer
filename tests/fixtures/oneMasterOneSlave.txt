		  402 Connect	phpMasterGeneric@192.168.1.1 on myDb
		  402 Query	SET NAMES UTF8, time_zone = "+0:00"
		  402 Query	SELECT * FROM customer WHERE idCustomer = 1
		  403 Connect	phpSlaveGeneric@192.168.1.2 on myDb
		  403 Query	SET NAMES UTF8, time_zone = "+0:00"
		  403 Query	SELECT * FROM customer
		  403 Query	SELECT email
                    FROM customer
                    WHERE idCustomer = 1
