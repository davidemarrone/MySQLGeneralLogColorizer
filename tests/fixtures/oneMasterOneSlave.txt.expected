[0;94m		  402 Connect	phpMasterGeneric@192.168.1.1 on myDb
[0m[0;31m		  402 Query	SET NAMES UTF8, time_zone = "+0:00"
[0m[0;31m		  402 Query	SELECT * FROM customer WHERE idCustomer = 1
[0m[0;94m		  403 Connect	phpSlaveGeneric@192.168.1.2 on myDb
[0m[0;32m		  403 Query	SET NAMES UTF8, time_zone = "+0:00"
[0m[0;32m		  403 Query	SELECT * FROM customer
[0m[0;32m		  403 Query	SELECT email
[0m[0;32m                    FROM customer
[0m[0;32m                    WHERE idCustomer = 1
[0m